src/components/Space-missions.js

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { v4 as uuidv4 } from "uuid";
+import Table from "react-bootstrap/Table";
+import { joinMission, getMissions, leaveMission } from "../redux/Missions";
+
+const selectMissions = (state) => state.missions;
+const Missions = () => {
+  const selectedMissions = useSelector(selectMissions);
+  const dispatch = useDispatch();
+  const getMissionsData = () => {
+    if (selectedMissions.length === 0) {
+      dispatch(getMissions());
+    }
+  };
+  useEffect(() => {
+    getMissionsData();
+  }, []);
+  return (
+    <div>
+      <Table striped bordered hover size="sm" className="table">
+        <thead>
+          <tr>
+            <th> Mission </th> <th> Description </th> <th> Status </th>{" "}
+            <th> {} </th>{" "}
+          </tr>{" "}
+        </thead>{" "}
+        <tbody>
+          {" "}
+          {selectedMissions.map((m) => (
+            <tr key={uuidv4()}>
+              <td className="mission-name"> {m.mission_name} </td>{" "}
+              <td> {m.mission_description} </td>{" "}
+              <td className="status" width="100px">
+                {" "}
+                {m.isReserved ? (
+                  <div
+                    className="primary"
+                    bg="success"
+                    style={{ backgroundColor: "blue", width: "130px" }}
+                  >
+                    {" "}
+                    ACTIVE MEMBER{" "}
+                  </div>
+                ) : (
+                  <div
+                    className="secondary"
+                    bg="secondary"
+                    style={{ backgroundColor: "grey", width: "130px" }}
+                  >
+                    {" "}
+                    NOT A MEMBER{" "}
+                  </div>
+                )}{" "}
+              </td>{" "}
+              <td className="status" width="150px">
+                {" "}
+                {m.isReserved ? (
+                  <button
+                    type="button"
+                    className="join"
+                    style={{ border: "1px solid red", width: "130px" }}
+                    variant="outline-danger"
+                    onClick={() => dispatch(leaveMission(m.mission_id))}
+                  >
+                    {" "}
+                    Leave Mission{" "}
+                  </button>
+                ) : (
+                  <button
+                    type="button"
+                    className="leave"
+                    variant="outline-dark"
+                    style={{ border: "1px solid grey", width: "100px" }}
+                    onClick={() => dispatch(joinMission(m.mission_id))}
+                  >
+                    {" "}
+                    Join Mission{" "}
+                  </button>
+                )}{" "}
+              </td>{" "}
+            </tr>
+          ))}{" "}
```

Comment:
- [ ] For this particular project, the React conditional render syntax is required for conditionally displaying both the Rockets and the Missions:

![image](https://user-images.githubusercontent.com/57421823/151388114-547dfc15-0380-44e6-b274-b7f7fbe510bb.png)

Don't worry! I'll show you how!! 💪 💪 

This is an example:

![image](https://user-images.githubusercontent.com/57421823/151388576-018db458-f6d7-4bfc-b0f9-a90dd7768f4f.png)

Another example with a kind of "if/else" scheme:

![image](https://user-images.githubusercontent.com/57421823/151396150-0479bcef-bb3b-44d2-b1a7-e696d222f7d2.png)


The rule is that if the condition on the left of the `&&` is true, the action of the right will be run. 💪 

Now that you know how to do it kindly please implement this syntax to your missions and rockets conditional rendering (i.e. for the "reserved" badge, the "join" badge, etc). ✔️